.github/workflows/build.yml

```diff
@@ -61,38 +54,29 @@ jobs:
             path: ~/.nuget/packages
             # We don't use a lockfile, so hash all files where we might be keeping <PackageReference> tags
             # Workloads also get installed in the nuget packages cache, so if you modify workloads below, increment the number in the cache keys
-            key: ${{ runner.os }}-nuget+workloads2-${{ hashFiles('**/*.*proj', '**/*.props') }}
-            restore-keys: ${{ runner.os }}-nuget+workloads2
+            key: ${{ runner.os }}-nuget+workloads3-${{ hashFiles('**/*.*proj', '**/*.props') }}
+            restore-keys: ${{ runner.os }}-nuget+workloads3
 
-        # .NET Android workloads don't support Linux https://github.com/dotnet/sdk/issues/22411
+        - name: Install .NET Workloads (Ubuntu)
+          if: startsWith(matrix.os, 'ubuntu')
+          run: 'dotnet workload install maui-android --temp-dir "${{ runner.temp }}"'
         - name: Install .NET Workloads (Windows)
           if: startsWith(matrix.os, 'windows')
-          run: dotnet workload install maui-android maui-windows
+          run: 'dotnet workload install maui-android maui-windows --temp-dir "${{ runner.temp }}"'
         - name: Install .NET Workloads (macOS)
           if: startsWith(matrix.os, 'macos')
-          run: dotnet workload install maui-android maui-ios maui-maccatalyst
+          run: 'dotnet workload install maui-android maui-ios maui-maccatalyst --temp-dir "${{ runner.temp }}"'
 
         - name: Build
-          run: dotnet build ${{ matrix.slnf }} -c Release /p:CopyLocalLockFileAssemblies=true
+          run: dotnet build -c Release /p:CopyLocalLockFileAssemblies=true
```

Comment:
Could we set this property here: https://github.com/getsentry/sentry-dotnet/blob/main/Directory.Build.props#L20-L22

This way we don't need to remember to pass it in different places: https://github.com/getsentry/sentry-dotnet/search?q=p%3ACopyLocalLockFileAssemblies

Reply:
No, because we don't want to do that in the CodeQL build.